### ISSUE TITLE
Support multipart responses or event streams

### ISSUE THREAD

**Issue by babichjacob** [svelte-add/graphql#1]
`graphql-helix`'s [README shows an example for Express(-like) servers](https://github.com/contrawork/graphql-helix/#basic-usage). We meet it 1:1 until the `else if (result.type === "MULTIPART_RESPONSE") {` point.

In SvelteKit's Node adapter, the underlying server is actually made from the `http` library in Node, *and* we don't have access to the server in endpoints. 

So, everything that can be returned needs to be returned all together. 

Can we do anything to condense a multipart response or event stream?

**Comment by babichjacob** [comment]
Upstream issue: https://github.com/sveltejs/kit/issues/1563

**Comment by ibilux** [comment]
Any updates about this ?

**Comment by babichjacob** [comment]
> Any updates about this ?

See the linked issue, which has been very active recently.

**Comment by ibilux** [comment]
> See the linked issue, which has been very active recently.

Yeah I have been checking it, but to be honest I'm not familiar about streaming.
Is there any eta or a near solution ?

**Comment by babichjacob** [comment]
> Is there any eta or a near solution ?

It's up to there being an accepted resolution and implementation in SvelteKit core, but then I would need help updating this tool to make use of it—I don't personally know how GraphQL subscriptions or multipart responses work so I don't know how to implement or test them.

**Comment by ibilux** [comment]
@babichjacob we will try to implement it when they accepted and merge it.

**Comment by omar-shahid** [comment]
Is this project still alive? I would like to help with this issue.
